oak/server/rust/oak_runtime/src/node/grpc_server.rs

```diff
@@ -208,6 +213,8 @@ impl GrpcServerNode {
     /// channel to the [`GrpcServerNode::channel_writer`].
     /// Returns a [`Handle`] for reading a gRPC response from.
     fn process_request(&self, request: GrpcRequest) -> Result<Handle, GrpcServerError> {
+        HTTP_COUNTER.inc();
```

Comment:
Is it a Singleton counter?

Reply:
This is a static variable. So we I guess we could have other counters too. I don't know what is the best practice for using these metrics. I'll have to investigate that part.